regulation-worker/internal/delete/api/api_test.go

```diff
@@ -670,6 +776,37 @@ func (cpRespProducer *cpResponseProducer) mockCpRequests() *chi.Mux {
 			return
 		}
 	})
+
+	srvMux.HandleFunc("/workspaces/{workspaceId}/destinations/{destinationId}/authStatus/toggle", func(w http.ResponseWriter, req *http.Request) {
+		if req.Method != http.MethodPut {
+			w.WriteHeader(http.StatusNotFound)
+			return
+		}
+		// iterating over request parameters
+		for _, reqParam := range []string{"workspaceId", "destinationId"} {
+			param := chi.URLParam(req, reqParam)
+			if param == "" {
+				// This case wouldn't occur I guess
+				http.Error(w, fmt.Sprintf("Wrong url being sent: %v", reqParam), http.StatusInternalServerError)
+				return
+			}
+		}
+
+		cpResp := cpRespProducer.GetNext()
+		// sleep is being used to mimic the waiting in actual transformer response
+		if cpResp.timeout > 0 {
+			time.Sleep(cpResp.timeout)
+		}
+		w.Header().Set("Content-Type", "application/json")
+		w.WriteHeader(cpResp.code)
+		// Lint error fix
+		_, err := w.Write([]byte(cpResp.response))
+		if err != nil {
+			fmt.Printf("I'm here!!!! Some shitty response!!")
```

Comment:
```suggestion
```

Reply:
@sanpj2292 this is still here :slightly_smiling_face:

Reply:
I am searching in my IDE, this is not found. Not sure if I'm missing something here 😅